modules/rosa-sts-aws-requirements.adoc

```diff
@@ -21,6 +21,100 @@ The following prerequisites must be complete before you deploy a {product-title}
 Customers are encouraged, but not mandated, to deploy resources in a Virtual Private Cloud (VPC) separate from the VPC hosting {product-title} and other Red Hat supported services.
 ====
 
+//
+// Start work for OSDOCS-3198
+//
+[id="rosa-associating-account_{context}"]
+== Associating your AWS account
+
+Performing {product-title} (ROSA) cluster provisioning tasks in {cluster-manager} require linked `ocm-role` and `user-role` resources to your AWS account by using your Amazon Resource Name (ARN).
+
+.Prerequisites
+
+* You have an AWS account.
+* You are using {cluster-manager-url} to create clusters.
+* You have the permissions required to install AWS account-wide roles.
+* You have installed and configured the latest AWS (`aws`) and ROSA (`rosa`) CLIs on your installation host.
+* You have created your `ocm-role` and `user-role` IAM roles.
+
+.Procedures
+
+1. From your desired CLI, link your `ocm-role` resource to your AWS account by using your Amazon Resource Name (ARN):
++
+[NOTE]
+====
+You must have organization administrator privileges to run the `rosa link` command. After you link the `ocm-role` resource with your AWS account, it is visible for all users in the organization.
+====
++
+[source,terminal]
+----
+rosa link ocm-role --role-arn <arn>
+----
++
+.Example output
+[source,terminal]
+----
+I: Linking OCM role
+? Link the '<AWS ACCOUNT ID>` role with organization '<ORG ID>'? Yes
+I: Successfully linked role-arn '<AWS ACCOUNT ID>' with organization account '<ORG ID>'
+----
+1. From your desired CLI, link your `user-role` resource to your AWS account by using your Amazon Resource Name (ARN):
```

Comment:
The same applies here. I think that the linkage is from AWS where the IAM role was created, to the Red Hat user account. Please confirm this with the SME team and if this is correct, please apply this change throughout the PR. Thanks!